python/test/CMakeLists.txt

```diff
@@ -237,6 +237,7 @@ ot_pyinstallcheck_test (NonStationaryCovarianceModelFactory_std)
 ot_pyinstallcheck_test (ProductCovarianceModel_std)
 ot_pyinstallcheck_test (StationaryFunctionalCovarianceModel_std IGNOREOUT)
 ot_pyinstallcheck_test (LatentVariableModel_std IGNOREOUT)
+ot_pyinstallcheck_test (ProcessAndFieldOperators_std IGNOREOUT)
```

Comment:
should be named "ProcessSample_Field_operators" (without std suffix)
or even better could be split accross 2 files: ProcessSample_operators & Field_operators